project/citation.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Citation
+---
+
+{% include citation.md %}
```

Comment:
Is there a reason for using an include instead of just using its contents? This way it can be reused off course.

Reply:
It's included in two places: the front page and its own page.

I made a separate page for it for direct linking (from the footer) because when I tried linking to the anchor (e.g. `/#citation`) the page didn't scroll to the correct place, possibly because some elements on the front page are loaded dynamically after the initial page load.

Reply:
oh I missed the second use of the include. ok! https://bedroesb.github.io/about#citation this works for me though (without the backslash)

Reply:
For me it scrolls here:
![image](https://user-images.githubusercontent.com/503373/159233841-9adf71ab-7029-4237-b619-82d2f9bb658e.png)
unless I'm already on the page when visiting the anchor